plugins/10_Dragonflight/zones/thaldraszus.lua

```diff
@@ -618,3 +618,67 @@ map.nodes[34676541] = HemetNesingwaryJr({
 map.nodes[50674562] = HemetNesingwaryJr({
     rewards = {Achievement({id = 16542, criteria = 55702})}
 }) -- Northern Thaldraszus Hunt
+
+-------------------------------------------------------------------------------
+-------------------------- ONE OF EVERYTHING, PLEASE --------------------------
+-------------------------------------------------------------------------------
+
+val.nodes[60921096] = Collectible({
+    label = '{item:200904}',
+    icon = 237358,
+    note = L['picante_pomfruit_cake_note'],
+    group = ns.groups.SPECIALTIES,
+    rewards = {Achievement({id = 16621, criteria = 55733})}
+}) -- Picante Pompfruit Cake
+
+val.nodes[29046503] = Collectible({
+    label = '{item:201045}',
+    icon = 133994,
+    note = L['icecrown_bleu_note'],
+    group = ns.groups.SPECIALTIES,
+    rewards = {Achievement({id = 16621, criteria = 55931})}
+}) -- Icecrown Bleu
+
+map.nodes[50054267] = Collectible({
+    label = '{item:201046}',
+    icon = 132802,
+    note = L['dreamwarding_dripbrew_note'],
+    group = ns.groups.SPECIALTIES,
+    rewards = {Achievement({id = 16621, criteria = 55933})}
+}) -- Dreamwarding Dripbrew
+
+local ArcanostabilizedProvisions = Collectible({
+    label = '{item:201047}',
+    icon = 134043,
+    note = L['arcanostabilized_provisions_note'],
+    group = ns.groups.SPECIALTIES,
+    rewards = {Achievement({id = 16621, criteria = 55934})}
+}) -- Arcanostabilized Provisions
+
+local tpf = Map({id = 2085, settings = false}) -- The Primalist Future
+tpf.nodes[61555322] = ArcanostabilizedProvisions
+map.nodes[59828223] = ArcanostabilizedProvisions
```

Comment:
This probably won't be the last thing we add to the primalist future map. I would move this variable def up to the top with `map` and `val`.

For the node here, try adding it to just the `tpf` map with `parent = map.id` and see if the coords translate. If they do not, we'll have to leave it as two separate coords here.

Reply:
I'll give it a shot and see. I believe they are completely disconnected zones.

Reply:
What do you know, they are connected!